examples/support/mock_data.py

```diff
@@ -120,13 +143,35 @@ def generate_galaxy_catalog(cluster_m, cluster_z, cluster_c, cosmo, ngals, Delta
     galaxy_catalog['id'] = np.arange(ngals)
     return galaxy_catalog
 
+def _chang_distrib(z):
+    """
+    A private function that returns the Chang et al (2013) galaxy redshift distribution function
+    """
+    alpha, beta, z0 = 1.24, 1.01, 0.51
+    return (z**alpha)*np.exp(-(z/z0)**beta)
```

Comment:
The docstrings now explicitly mention that this function is unnormalized. I'd rather limit docstrings to tell what the function is or does and not expend to what something it will be used for eslewhere in the code.